src/osd/scrubber/scrub_backend.h

```diff
@@ -387,7 +401,7 @@ class ScrubBackend {
 
   // shorthands:
   ConfigProxy& m_conf;
-  LogChannelRef clog;
+  LoggerSinkSet* clog;
```

Comment:
And store as a reference here.

Reply:
That's a problem:
For example: take

LoggerSinkSet* PgScrubber::get_logger() const { return m_osds->clog.get(); }

I can't return a non-owning ref here (LoggerSinkSet&). And will have similar issues if
returning a owning ref to base.

Reply:
https://github.com/athanatos/ceph/commit/30736134ce9dc56126b472a4338e6b0332e00e1f I think we can return it as a reference.  It's not *that* important, but I'd prefer to use references in non-owning structs if possible.

Reply:
done